src/components/__test__/CalculatorPage.test.js

```diff
@@ -0,0 +1,7 @@
+import renderer from 'react-test-renderer';
+import CalculatorPage from '../CalculatorPage';
+
+it('Render the Calculator page user interaction', () => {
+  const tree = renderer.create(<CalculatorPage />).toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] As required for the project, kindly ensure that there are the usage of React Testing Library to simulate user interaction.